src/pages/__tests__/Dashboard.test.js

```diff
@@ -42,18 +42,35 @@ describe('<Dashboard>', () => {
   })
 
   expectedStrings.map(str => {
-    test(`renders ${str} on page`, () => {
-      const $ = cheerio.load(
-        render(
-          html`
-            <${Dashboard} data=${data} />
-          `,
-        ),
-      )
-      expect($('h1').text()).toEqual('Hi, Fred')
+    //If we have an object inside our data object (for example income info) we need to loop thru it and not just process the top most value
```

Comment:
We could also make the top-level object into:
```
  {
    name: 'Fred Smith',
    address: 'Mississauga',
    maritalStatus: 'Married',
    children: 5,
    employerName: 'Blorb Corp',
    year: 2019,
    box12: '321987645',
    box14: 10000,
    box22: 1000,
  },
```

to keep our original loop, since we're still expecting the string to be on the page. But if we want to match the "user object" we have, then we can keep this as it is now.

Reply:
>if we want to match the "user object" we have, then we can keep this as it is now.

I think we want to match it as closely as possible so our test are as close to "real life" as possible.